src/app/core/services/navigation/group-nav-tree.service.ts

```diff
@@ -23,14 +23,24 @@ export class GroupNavTreeService extends NavTreeService<GroupInfo> {
     super(currentContent);
   }
 
+  childrenNavigation(): OperatorFunction<GroupInfo|undefined,GroupNavigationData|undefined> {
+    return pipe(
+      distinctUntilChanged((g1, g2) => g1?.route.id === g2?.route.id),
+      switchMap(group => {
+        if (!group || group.route.isUser) return of(undefined);
+        return concat(of(undefined), this.groupNavigationService.getGroupNavigation(group.route.id));
+      })
+    );
+  }
+
```

Comment:
> Btw, the logic `if (…) return of(undefined); else return concat(of(undefined), …)` looks shareable

Not really... or at least not a operator to apply just before or just after as it would be useful. The cleanest way would be to create an operator wrapper the switchmap, but isn't it overkill ?